assets/xml/objects/object_ik.xml

```diff
@@ -18,54 +18,54 @@
         <Animation Name="object_ik_Anim_0008DC" Offset="0x8DC"/>
         <Animation Name="object_ik_Anim_000C6C" Offset="0xC6C"/>
```

Comment:
Are you able to fill out more of this object based on what you did in MM?

Reply:
This animations appear to be OoT specific, when loading them in z64utils it has "Animation is glitchy, to view this animation load it in-game", so I'll do some more research to see if I can figure it out

Reply:
If they are OoT specific I'll leave it up to you if you want to document them.
Note if z64utils says they are glitchy, you will need to mod the game and apply them manually in game. Usually they are quite chaotic.